spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-local/src/main/java/org/springframework/cloud/data/module/deployer/local/LocalModuleDeployer.java

```diff
@@ -57,10 +57,17 @@ public LocalModuleDeployer(ModuleLauncher launcher) {
 	@Override
 	public ModuleDeploymentId deploy(ModuleDeploymentRequest request) {
 		String module = request.getCoordinates().toString();
+		if (request.getCount() != 1) {
+			logger.warn(String.format("%s only supports a single instance per module; ignoring count=%d for %s",
```

Comment:
Why is that really? I get it because of _e.g._ port conflicts (and someone convinced me that having 8080 as a default was a good idea. And I agreed :) ), but apart from that there is really nothing stopping us from having several instances in the same process, is there? I mean, the binder would bind correctly, etc